### DISCUSSION TITLE
Metronome should play from music channel, not effects

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#28662] [Ideas]
currently, if the metronome is off time for a map (which it often is, which is a separate issue) then you simply cant play that map with the muted mod. The metronome should instead play from the music channel so that the offset of the metronome can be adjusted. (this also would allow adjusting the volume of the metronome.)

**Comment by peppy** [comment]
I think you may be a bit confused. Adjusting the offset *will* change the metronome. It has to.

**Reply by sparkeynk** [reply]
Adjusting the offset actually does not change the metronome's offset in the muted mod for me as well. As the original post says, the metronome's volume seems to be linked to the Effect's volume and not the Music's volume which could be causing this issue. Anyways I think most people would prefer to be able to adjust the metronome's volume separately from the hitsounds.

**Comment by FibreTTP** [comment]
I think it should play from music so you can change its volume without affecting hitsound volume.
